README.md

```diff
@@ -149,14 +152,63 @@ qtpy pyright-config
 ```
 
 you will get the necessary configs to be included in your project files. If you don't
-have them, it is recommended to create the latter.
+have them, it is recommended to create the latter. For example, in an environment where PyQt5 is installed and selected
+(or the default fallback, if no binding can be found in the environment),
+this would output the following:
```

Comment:
```suggestion
have them, it is recommended to create the latter. For example, in an environment where PyQt5
is installed and selected (or the default fallback, if no binding can be found in the
environment), this would output the following:
```